Samples/Hast.Samples.Consumer/SampleRunners/ImageSharpSampleRunner.cs

```diff
@@ -0,0 +1,58 @@
+using Hast.Layer;
+using Hast.Samples.SampleAssembly;
+using Hast.Samples.SampleAssembly.ImageSharpModifications.Resize;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Processing;
+using System.Threading.Tasks;
+
+namespace Hast.Samples.Consumer.SampleRunners
+{
+    internal class ImageSharpSampleRunner : ISampleRunner
+    {
+        public void Configure(HardwareGenerationConfiguration configuration)
+        {
+            configuration.AddHardwareEntryPointType<ImageSharpSample>();
+        }
+
+        public async Task Run(IHastlayer hastlayer, IHardwareRepresentation hardwareRepresentation, IProxyGenerationConfiguration configuration)
+        {
+            // In case you wish to test the sample with a larger file, the fpga.jpg file must be replaced. You can find
+            // a 100 megapixel jpeg here: https://photographingspace.com/100-megapixel-moon/
+            
+            // Not accelerated by hastlayer
+            RunSoftwareBenchmarks();
+
+            // Accelerated by hastlayer
+            using var image = Image.Load("fpga.jpg");
+
+            var resizeImage = await hastlayer
+                .GenerateProxy(hardwareRepresentation, new ImageSharpSample(), configuration);
+            var modifiedImage = resizeImage
+                .Resize(image, hastlayer, hardwareRepresentation.HardwareGenerationConfiguration);
+            modifiedImage.Save("fpga_resized_with_hastlayer_fpga.jpg");

```

Comment:
PascalCasing legyen inkább. Az "fpga_resized_wtih_modified_imagesharp"-ban amúgy typo is van.